doc/rose-rug-suites-II.html

```diff
@@ -377,12 +377,10 @@ <h3 class="alwayshidden">Environment (2)</h3>
                 <h3 class="alwayshidden">Environment (3)</h3>
 
                 <ul class="incremental">
-                  <li><var>ROSE_TASK_LOG_ROOT</var>, the name of
-                  the task including the cycle time - if you prefix
-                  a filename with <samp>$ROSE_TASK_LOG_ROOT</samp>
-                  in the <samp>log/job/</samp> directory, it will
-                  be associated with the task in the suite log
-                  viewer.</li>
+                  <li><var>ROSE_TASK_LOG_ROOT</var>, the path where
+                  log files for a task are stored - if you create
```

Comment:
Should be "the path to log files for a task without the extension" - as this goes to:

ROSE_TASK_LOG_ROOT=$HOME/cylc-run/suite/log/job/1/task/01/job

rather than:

ROSE_TASK_LOG_ROOT=$HOME/cylc-run/suite/log/job/1/task/01

which is what the above reads like (which is ROSE_TASK_LOG_DIR)